modules/update-changing-update-server-cli.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster-rhel-compute.adoc
+
+[id="update-changing-update-server-cli_{context}"]
+= Changing the update server by using the CLI
+
+Changing the update server is optional. If you have an OpenShift Update Service (OSUS) installed and configured locally, you must set the URL for the server as the `upstream` to use the local server during updates. The default value for `upstream` is \https://api.openshift.com/api/upgrades_info/v1/graph.
```

Comment:
Since \https://api.openshift.com/api/upgrades_info/v1/graph is an [example URL](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls), please add backticks:

`\https://api.openshift.com/api/upgrades_info/v1/graph`